### PR TITLE
Allow inclusion of hidden fields on table metadata

### PR DESCRIPTION
**Pull request by robdaemon**
Adds a `include_hidden_fields` parameter to `/table/:id/query_metadata`
endpoint, so that you can choose if you want to include hidden fields in
the response.

### CODE REVIEW

**Comment by robdaemon** [comment]
This isn't the fix for the data model admin page visibility issue, but does make the API more complete.

**Comment by paulrosenzweig** [comment]
It might be good to have this merged before I finalize and merge #12513. @camsaul could you review it?

**Review by camsaul — APPROVED** [review]
Cool looks good to me. If something is waiting for review for a while you can ping me and I'll review it. I get a few dozen GitHub notifications a day so sometimes I end up missing something

**Pull request merged**
